### DISCUSSION TITLE
Flash Notification for Form Errors

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#2470] [Ideas]
When saving a form successfully, Filament sends a flash notification to alert you that the record was saved. However, when a form doesn't pass validation - Filament doesn't send a flash notification to alert you that there was a problem saving the record. On short forms, this isn't a huge problem since you most often can see the offending field that gets highlighted. But on long forms, the user has no idea that the form didn't save (unless they happen to notice that they never got the success flash notification).

I could swear that Filament used to do this, but I've checked several different recent versions and none do.

**Comment by danharrin** [comment]
Hi,

I don't think this was ever a Filament feature, but I think it would be cool.

Do you think it should happen by default?

**Reply by intrepidws** [reply]
I do, yes. But if you thought it would be better as an option, I would be fine with that.
